app/controllers/users.rb

```diff
@@ -3,6 +3,11 @@
   erb :'users/signup'
 end
 
+get '/users/:id' do
+  @user = User.find(params[:id])
```

Comment:
I usually use User.find_by(id: params[:id]) because it will fail gracefully (return nil) if the id is not found, as opposed to #find, which will through an erroron not found.